graphql_compiler/tests/test_data_tools/neo4j.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+
+from neo4j import GraphDatabase
+
+
+NEO4J_SERVER = 'localhost'
+NEO4J_PORT = 7687
+NEO4J_USER = 'neo4j'
+NEO4J_PASSWORD = 'root'
+
+
+class Neo4jClient:
```

Comment:
This won't work as you'd expect in Python 2, since it will create an "old-style" class. You should make it `Neo4jClient(object)` instead.